store/db.go

```diff
@@ -33,6 +34,9 @@ type DB struct {
 	dbdir string
 	lock  *lock.FileLock
 	sqldb *sql.DB
+	// In fact, the mutex here is not necessary, as we already
+	// use flocks. This is just for making race detector happy.
```

Comment:
No, it's not just for making the race detector happy, it's to prevent legitimate races!